src/common/types/store-data/assessment-result-data.ts

```diff
@@ -28,6 +29,7 @@ export interface AssessmentData {
     generatedAssessmentInstancesMap?: InstanceIdToInstanceDataMap;
     manualTestStepResultMap?: RequirementIdToResultMap;
     testStepStatus: ManualTestStatusData;
+    scanIncompleteWarnings?: ScanIncompleteWarningId[];
```

Comment:
Is there any particular reason for this being optional (here and other places)?

Reply:
It's intentional for it to be optional in the store data sections because it has to be backwards compatible with users that upgrade with old store data in place.